webapp/channels/src/components/settings_sidebar/settings_sidebar.tsx

```diff
@@ -110,17 +110,31 @@ export default class SettingsSidebar extends React.PureComponent<Props> {
             pluginTabList = (
                 <>
                     <hr/>
-                    <li
-                        key={'plugin preferences heading'}
-                        role='heading'
-                        className={'header'}
+                    <div
+                        aria-labelledby={'userSettingsModal.pluginPreferences.header'}
+                        role='region'
                     >
-                        <FormattedMessage
+                        <div
                             id={'userSettingsModal.pluginPreferences.header'}
-                            defaultMessage={'PLUGIN PREFERENCES'}
-                        />
-                    </li>
-                    {this.props.pluginTabs.map((tab, index) => this.renderTab(tab, index))}
+                            key={'plugin preferences heading'}
+                            role='heading'
+                            aria-level={3}
+                            className={'header'}
+                        >
+                            <FormattedMessage
+                                id={'userSettingsModal.pluginPreferences.header'}
+                                defaultMessage={'PLUGIN PREFERENCES'}
+                            />
+                        </div>
+                        <ul
```

Comment:
Having multiple tablists in the sidebar seems a bit odd to me, but I'm probably be overthinking how those should be used. I can't find anything online about how specifically to group items like we do here, so I'm good with this solution

Reply:
While trying to resolve a different issue. I found a better approach to structure the tablist by replacing the `<ul>` and `<li>` elements with `<div>` elements and added the appropriate roles and groupings. This approach allows us to use only one tablist. Here is the PR: [link](https://github.com/mattermost/mattermost/pull/29592) (This PR resolves both [MM-61648](https://mattermost.atlassian.net/browse/MM-61648) and [MM-61588](https://mattermost.atlassian.net/browse/MM-61588)).